standalone/detectors/fasttrack/include/PrefixTreeDepot.h

```diff
@@ -0,0 +1,278 @@
+
+#ifndef TREEDEPOT_HEADER_H
+#define TREEDEPOT_HEADER_H 1
+#pragma once
```

Comment:
Please remove insignificant `1` and the redundant pragma since include guards already exist and are used 
```suggestion
#define TREEDEPOT_HEADER_H
```